goss-testing/tests/ncn/goss-check-default-gw.yaml

```diff
@@ -0,0 +1,41 @@
+#
```

Comment:
This test seems to be covered by the existing `goss-default-gateway-points-to-CMN-gateway.yaml` test. That test is more extensive, in that it not only verifies that there is a default route, but also validates that it is the correct default route. Is there a reason to create a new test which just does this part?